en/asset/audio.md

```diff
@@ -1,29 +1,27 @@
-# Audio
+# Audio Assets
 
-__Sound__ assets are __audio files__. An __audio system__ has two main functions: __playing background music__ and __playing short sound effects__.
-For sound assets, there is no difference between the two. After all, audio assets are imported into the editor, `AudioClip` assets perform related audio operations through the `AudioSource` system component. To use the audio system, please refer to the [Audio System](../audio-system/overview.md) documentation.
+An **audio** asset is an **audio file**.<br>
+For the **audio system**, its interface is geared to two main needs: "music" of longer lengths, and "sound" effects of shorter lengths.<br>
+But for audio assets, there is no difference between the two. After all audio assets are imported into the editor, AudioClip assets perform the relevant audio operations through the AudioSource audio system component. For more information on using the audio system, please refer to the [Audio System](../audio-system/overview.md) documentation.
 
-## Supported audio asset formats
+## Supported Formats of Audio Assets
 
-Currently, the engine's audio system can support the following formats:
+The current audio system of the engine is able to support the formats natively supported by the web:
+- .ogg
+- .mp3
+- .wav
+- .mp4
+- .m4a

```

Comment:
```suggestion
- `.m4a`

```

Reply:
our CONTRIBUTIONS.md technically uses backticks for file names and extensions. What do you think?